packages/gui/src/components/offers2/CreateOfferBuilder.tsx

```diff
@@ -71,13 +71,14 @@ export default function CreateOfferBuilder(props: CreateOfferBuilderProps) {
 
   const handleSubmit = useCallback(
     async (values: OfferBuilderData) => {
-      const { assetsToUnlock, ...localOffer } = await offerBuilderDataToOffer(
-        values,
+      const { assetsToUnlock, ...localOffer } = await offerBuilderDataToOffer({
+        data: values,
         wallets,
-        offers || [],
-        false,
-        true
-      );
+        offers: offers || [],
+        validateOnly: false,
+        considerNftRoyalty: true,
+        allowEmptyOfferColumn: false,
```

Comment:
Is initiating one-sided offer not allowed?

Reply:
Initiating a one-sided offer is allowed when creating a new offer, but that one-sided offer *MUST* be offering some asset. We don't support creating offers that only request assets.

Maybe that will be a feature that we support at some point, but currently the intent is to only support creating an offer if some assets are being offered.

Reply:
Ok, thanks for the detail